### DISCUSSION TITLE
Erzeuger via SAE und Smartmeter in SMA-Welt integrierbar?

### DISCUSSION THREAD

**Discussion by AchimWilden** [camueller/SmartApplianceEnabler#444] [Q&A]
Hallo,

dass man mit dem SAE wunderbar Verbrauchen bzw deren Verbrauchsdaten in die SMA Welt hieven kann, weiß man ja...
Wie siehts denn mit Erzeugern aus? 
Konkret: 
Ich habe eine Anlage mit SHM WR und ... alles von SMA. Diese Anlage will ich nun um einen WR erweitern, der nicht von SMA ist.
Wenn ich das einfach so mache, geht das natürlich technisch aber die Visualisierung im SunnyPortal ist kaputt. Mein Haus könnte einen negativen Verbrauch haben, wenn der tatsächliche Verbrauch kleiner ist als die durch den "Fremd"-WR eingespeiste Leistung.
In anderen Foren habe ich schon gelesen, dass die Verbrauchsanzeige auf 0W sinkt aber nie negativ wird.
Alles in allem wird die Anzeige im SunnyPortal dadurch recht unintuitiv.
Die SMA Lösung wäre der kauf eines Smart Energy Meters, neu für schlanke 400 Steine.
Da möchte ich drumherum kommen und kam wieder zu der Idee den SAE hier als Verbindungsglied für "preiswertere" Smartmeter und SMA zu nutzen. 
Weiß jemand ob ich über den Weg zum Ziel komme?

Viele Grüße Achim

**Comment by camueller** [comment]
Der SAE implementiert das SEMP-Protokoll von SMA. Dieses unterstützt keine Erzeuger.

**Reply by AchimWilden** [reply]
Das hab ich befürchtet. Schade. Danke für die Antwort.
